docs/newbs_flashing.md

```diff
@@ -216,11 +232,14 @@ If you have any issues with this, you may need to this:
     sudo make <my_keyboard>:<my_keymap>:avrdude
 
 
-Additionally, if you want to flash multiple boards, use the following command:
+#### Caterina commands
 
-    make <keyboard>:<keymap>:avrdude-loop
+There are a number of DFU commands that you can use to flash firmware to a DFU device:
 
-When you're done flashing boards, you'll need to hit Ctrl + C or whatever the correct keystroke is for your operating system to break the loop.
+* `:avrdude` - This is the normal option and waits until a caterina device is available, and then flashes the firmware. This will wait until it detects a new COM port, and then it will flash.
+* `:avrdude-loop` - This runs the same command as `:avrdude`, but after each device is flashed, it will attempt to flash again.  This is useful for bulk flashing. _This requires you to manually escape the loop by hitting Ctrl+C, Cmd+C, or whatever the hotkey is for your system._
```

Comment:
```suggestion
* `:avrdude-loop` - This runs the same command as `:avrdude`, but after each device is flashed, it will attempt to flash again.  This is useful for bulk flashing. _This requires you to manually escape the loop by hitting Control+C._
```